src/pages/txp-details/txp-details.scss

```diff
@@ -60,6 +71,33 @@ page-txp-details {
   .container-from {
     .wallet,
     .payment-proposal-to {
+      justify-content: space-between;
+      .address-divider {
```

Comment:
It is used in receive, confirm, tx-details and txp.. I think we can move it to app.scss, to prevent duplicate code